INTMDB-285: Fix org_invitations issue (#643)
mongodb/terraform-provider-mongodbatlas: website/docs/r/org_invitation.html.markdown

* `invitation_id` - Unique 24-hexadecimal digit string that identifies the invitation in Atlas.
* `inviter_username` - Atlas user who invited `username` to the organization.

> > **NOTE**: If the user declines the invitation or accepts and leave the organization, terraform provider will send again the invitation if you don't remove the invitation from the resource definition.

Reviewer: ```suggestion
> **NOTE**: If the user declines the invitation or accepts the invitation and then leaves the organization, the  provider will re-add the invitation the invitation if the resource definition is not removed from the Terraform configuration. 
```

Author: done!

* `invitation_id` - Unique 24-hexadecimal digit string that identifies the invitation in Atlas.
* `inviter_username` - Atlas user who invited `username` to the organization.

> > **NOTE**: If the user declines the invitation or accepts the invitation and then leaves the organization, the  provider will re-add the invitation the invitation if the resource definition is not removed from the Terraform configuration.
> 
> > **NOTE**: The provider also will validate 3 cases:
> >* If the user haven't accepted the invitation, nothing happens.
> >* If the user already belongs to the organization, the provider will remove the invitation from the state, but it's necessary to remove from the terraform resource configuration.
> >* If the user is not a member of the organization or rejected the invitation, the provider will try to send again the invitation (unless the invitation get removed from the terraform resource configuration). 
> 

Reviewer: In general I think this is headed in the right direction but might still be a bit confusing for users.  I just ran through an invitation process and a user cannot reject an invitation.  They can not accept but only the sender of the invitation can delete it.   Taking your idea and just editing it a bit to what I hope will be clearer: 

```suggestion
> **NOTE**: Possible provider behavior depending on the invitee's action:
>* If the user has not yet accepted the invitation, the provider leaves the invitation as is.
>* If the user has accepted the invitation and is now an organization member, the provider will remove the invitation from the Terraform state.  The invitation must then be removed from the Terraform resource configuration.
>* If the user accepts the invitation and then leaves the organization, the provider will re-add the invitation if the resource definition is not removed from the Terraform configuration.

```

Author: Done! and thank you for the suggestion :)
